client/bin/multi-client.js

```diff
@@ -337,18 +337,29 @@ class State {
 					: 1);
 
 			if (yargs.print) {
-				children.forEach(child => {
+				Object.values(children).forEach(child => {
 					console.log(`=====| ${child.outputPath}`);
-					console.log(fs.readFileSync(child.outputPath));
```

Comment:
note that I don't know what that is even trying to achieve. in my testing this file never exists...